core/src/com/unciv/logic/GameStarter.kt

```diff
@@ -167,6 +169,18 @@ object GameStarter {
     private fun addCivTechs(gameInfo: GameInfo, ruleset: Ruleset, gameSetupInfo: GameSetupInfo) {
         for (civInfo in gameInfo.civilizations.filter { !it.isBarbarian() }) {
 
+        //Catch uniques from techs that should've triggered already
```

Comment:
Where are we adding these 'untriggered techs' in the first place, if not through addTechnology?

Reply:
They're added in `addCivilizations`, line 353 (or 367 of this PR). I did first immediately think to move those added techs to addCivTechs to allow for using addTechnology but then that adds in the notification that you've reached that era. It probably is safer to find a way to move it to here anyways, but then we either need to accept that it shows we made it to the ancient era on game start or unless we rework how addTechnology checks that